changelog/unreleased/change-oc-breadcrumb-a11y

```diff
@@ -0,0 +1,8 @@
+Change: Advance accessibility to oc-breadcrumb component
```

Comment:
Could you rewrite the changelog item in a way that it reflects `what we did` instead of `what we will do` please? :-) Like `For enhanced accessibility of the oc-breadcrumb component we wrapped it in a <nav> tag.` etc.